Package.swift

```diff
@@ -28,7 +28,8 @@ let package = Package(
                 .product(name: "CombineExt", package: "CombineExt"),
                 .product(name: "KeychainAccess", package: "KeychainAccess")
             ],
-            path: "./Sources/GoodPersistence"
```

Comment:
Toto si koli comu vyhadzoval?

Reply:
Iba som to dal do korena, lebo som pridal PrivacyInfo file na uroven Goodpersistance foldera. Nechcel som to davat priamo ku zdrojakom.